packages/loading/app/app.vue

```diff
@@ -147,6 +147,12 @@ export default {
       this.onData(data)
     },
 
+    formatTip () {
+      const tip = this.options.tip.text
```

Comment:
This way we can also be sure there is nothing vulnerable passed to `v-html` regardless of data source :) 

```suggestion
      const tip = this.options.tip.text.replace(/(<([^>]+)>)/gi, "")
```

Reply:
ok added